app/views/games/_captured-display.html.erb

```diff
@@ -0,0 +1,8 @@
+<h5>Captured Pieces</h5>
```

Comment:
This Captured Piece H5 text seems to be duplicated on the show page and seemed to appear and disappear while moving pieces around.  Let's remove either this text in your partial or on the show page.

![duplicate](https://user-images.githubusercontent.com/20831001/33112921-8bad4bf4-cf0a-11e7-9342-e1a6b35378e2.jpg)